validator/metrics/collector.go

```diff
@@ -85,6 +86,18 @@ func (c *validatorsCollector) Collect() ([]string, error) {
 				runningIbftsCountValidator, hex.EncodeToString(pk), runningIbftsValidator))
 		}
 
+		if v, exist := c.validatorCtrl.GetValidator(pubKey.SerializeToHexStr()); exist {
```

Comment:
@nivBlox not sure it's relevant, might be better to separate to a different PR

Reply:
@amir-blox is on it